### ISSUE TITLE
Don't name-resolve `extern fn`s

### ISSUE THREAD

**Issue by PatchMixolydic** [PatchMixolydic/laria#10]
Instead, have them just resolve to their idents when lowered. For instance:

```rust
mod native {
    extern fn print(s: string);
}

fn main() {
    native::print("Hello");
}
```

would lower to something like
```asm
::root::main:
    push "print"
    get_global ; stack now contains `NativeFn(native_print)`
    push "hello"
    jsr  1 ; call `native_print` with one argument, `"hello"`
```

The user could then insert a native fn:
```rust
vm.set_global("print", Value::NativeFn(native_print));
```

...which should bind to `print`, not `::root::print` as it does currently.

**Comment by PatchMixolydic** [comment]
Hmm... *usages* will have to be name-resolved, but ultimately, usages of them should end up with just their idents, rather than their full paths as with Laria functions. Confusing!

**Comment by PatchMixolydic** [comment]
Fixed by a8a14cd.
